skiko/src/jvmMain/kotlin/org/jetbrains/skiko/FPSCounter.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.skiko
+
+import java.util.*
+import kotlin.math.roundToInt
+
+internal class FPSCounter(
+    private val count: Int,
+    private val probability: Double
+) {
+    private var i = 0
+    private val times = LinkedList<Double>()
```

Comment:
Why `LinkedList' and not `DoubleArray` with sliding pointer?

Reply:
With DoubleArray we have to write our own sorted/average methods.

FPSCounter hasn't high overhead for FPS measuring purposes.